src/Pim/Bundle/CatalogBundle/Doctrine/Common/Saver/ProductSaver.php

```diff
@@ -56,12 +56,13 @@ public function save($product, array $options = [])
 
         $this->eventDispatcher->dispatch(StorageEvents::PRE_SAVE, new GenericEvent($product, $options));
 
-        $this->completenessManager->schedule($product);
+        // TODO: Temporary fix to allow the PIM to install. To remove once new storage works.
```

Comment:
can you add a ticket to this TODO (and all the followings) to do not forget them ?